module/documents/item.mjs

```diff
@@ -902,52 +902,27 @@ export default class Item5e extends SystemDocumentMixin(Item) {
    * @param {ItemUseOptions} [options]           Options used for configuring item usage.
    * @returns {Promise<ChatMessage|object|void>} Chat message if options.createMessage is true, message data if it is
    *                                             false, and nothing if the roll wasn't performed.
+   * @deprecated since DnD5e 4.0, targeted for removal in DnD5e 4.4
    */
   async use(config={}, options={}) {
-    if ( !this.isOwner ) {
-      ui.notifications.error("DND5E.DocumentUseWarn", { localize: true });
-      return null;
-    }
-    let item = this;
-    const is = item.system;
-    const as = item.actor.system;
-
-    // Ensure the options object is ready
-    options = foundry.utils.mergeObject({
-      configureDialog: true,
-      createMessage: true,
-      "flags.dnd5e.use": {type: this.type, itemId: this.id, itemUuid: this.uuid}
-    }, options);
-
-    // Define follow-up actions resulting from the item usage
-    if ( config.consumeSlotLevel ) {
-      console.warn("You are passing 'consumeSlotLevel' to the ItemUseConfiguration object, which now expects a key as 'slotLevel'.");
-      config.slotLevel = config.consumeSlotLevel;
-      delete config.consumeSlotLevel;
-    }
-    config = foundry.utils.mergeObject(this._getUsageConfig(), config);
-
-    /**
-     * A hook event that fires before an item usage is configured.
-     * @function dnd5e.preUseItem
-     * @memberof hookEvents
-     * @param {Item5e} item                  Item being used.
-     * @param {ItemUseConfiguration} config  Configuration data for the item usage being prepared.
-     * @param {ItemUseOptions} options       Additional options used for configuring item usage.
-     * @returns {boolean}                    Explicitly return `false` to prevent item from being used.
-     */
-    if ( Hooks.call("dnd5e.preUseItem", item, config, options) === false ) return;
-
-    // Are any default values necessitating a prompt?
-    const needsConfiguration = Object.values(config).includes(true);
-
-    // Display configuration dialog
-    if ( (options.configureDialog !== false) && needsConfiguration ) {
-      const configuration = await AbilityUseDialog.create(item, config);
-      if ( !configuration ) return;
-      foundry.utils.mergeObject(config, configuration);
+    foundry.utils.logCompatibilityWarning(
+      "The `Item5e#use` method has been deprecated and should now be called directly on the activity to be used.",
```

Comment:
I don't think there is such a thing as base activation on items. We could probably infer a default activity if the user isn't activating one specifically (e.g. if there is only a single activity on an item), but generally you need to trigger a specific activity.

For spells, I think they would all need to have an activity if they are to be cast. If you add a spell to your sheet without an activity we would probably show a preparation warning because all of the consumption logic is tied in with activities.

Reply:
Yeah, I'll stick that in the Todo list.